openmetrics/tests/conftest.py

```diff
@@ -35,12 +35,13 @@ def poll_mock():
     g3 = Gauge('metric3', 'memory usage', ['matched_label', 'node', 'timestamp'], registry=registry)
     g3.labels(matched_label="foobar", node="host2", timestamp="456").set(float('inf'))
 
+    data = ensure_unicode(generate_latest(registry))
+    data = data.replace("_total", "")
```

Comment:
This implies a breaking change, no?

Reply:
No, I don't think it does. It implies a breaking change from the prometheus producing metric side that we used in the tests, not in our consumer. Notice that the classes used to generate the data from `prometheus_client`.

Reply:
But does that mean users can also produce counter metrics with `_total` suffix and it won't work with this integration ?